packages/pwa-kit-dev/src/utils/script-utils.test.js

```diff
@@ -237,6 +237,8 @@ describe('scriptUtils', () => {
             expect(bundle.ssr_parameters).toEqual({})
             expect(bundle.ssr_only).toEqual(['ssr.js'])
             expect(bundle.ssr_shared).toEqual(['ssr.js', 'static/favicon.ico'])
+            expect(bundle.bundle_metadata).toHaveProperty('dependencies')
+            expect(bundle.bundle_metadata).toHaveProperty('cc_overrides')
```

Comment:
Accidentally setting `cc_overrides` to `undefined` would still pass this test. Might be worthwhile to change the assertions to test that `dependencies` is an object with some known key/value and `cc_overrides` is an array with some known value.